wrappers/macwrapper.cpp

```diff
@@ -555,8 +557,10 @@ static bool initializeZone(malloc_zone_t& zone) {
   zone.introspect   = NULL;
   zone.version      = 8;
   zone.memalign     = replace_malloc_zone_memalign;
+#if MAC_OS_X_VERSION_MAX_ALLOWED >= 1060 && !defined(__POWERPC__)
```

Comment:
Replace 1060 with `MAC_OS_X_VERSION_10_6`.

Reply:
Bump @barracuda156

Reply:
> Replace 1060 with `MAC_OS_X_VERSION_10_6`.

@emeryberger This will be wrong, since a macro is not defined prior to 10.6. Numerical constants work correctly though, since 1050 is defined on 10.5.

Reply:
Excellent point, my bad. I generally prefer macros / constants to magic numbers, but this is a good point. If you can toss in a note about the crazy pause instruction in old spinlock, I can merge.